apps/nextjs-website/src/components/HeroCounter.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+import MarkdownRenderer from './MarkdownRenderer';
+import { HeroCounter as HeroCounterRC } from '@react-components/components';
+import { HeroCounterProps } from '@react-components/types';
+import { HeroCounterSection } from '@/lib/fetch/types/PageSection';
+
+const makeHeroCounterProps = ({
+  subtitle,
+  background,
+  counterNumber,
+  counterText,
+  ...rest
+}: HeroCounterSection): HeroCounterProps => ({
+  ...rest,
+  ...(subtitle && { subtitle: MarkdownRenderer({ markdown: subtitle }) }),
+  counterNumber,
+  counterText,
+  ...(background && { background: background.url }),
+});
```

Comment:
```suggestion
const makeHeroCounterProps = ({
  subtitle,
  background,
  ...rest
}: HeroCounterSection): HeroCounterProps => ({
  ...rest,
  ...(subtitle && { subtitle: MarkdownRenderer({ markdown: subtitle }) }),
  ...(background && { background: background.url }),
});
```
Remove rendundancies. Collapse them into `...rest`.